src/Sparrow/Platform/Win32/Win32MemoryQueryMethods.cs

```diff
@@ -99,6 +99,35 @@ public static uint PageSize
             }
         }
 
+        private static string GetEncodedFilename(IntPtr processHandle, MEMORY_BASIC_INFORMATION memoryBasicInformation)
+        {
+            var filenameString = new byte[2048];
```

Comment:
Let's avoid allocating 2 KB on each call to this function. Use the buffer pool here as well